src/on-premises/install/quickstart/config.md

```diff
@@ -0,0 +1,329 @@
+# Configuration
+This lists all variables you can use to configure our playbook.
+
+## applications.yml
+### DNS configuration
+::: v-pre
+| Variables                   | Default value                            | Description                                            |
+|-----------------------------|------------------------------------------|--------------------------------------------------------|
+| app_dns_domain              | "domain.tld"                             | DNS (sub)domain use to build the app URLs.             |
+| api_dns_name                | `"api.{{ app_dns_domain }}"`             | DNS name use for the API URL.                          |
+| connect_dns_name            | `"connect.{{ app_dns_domain }}"`         | DNS name use for the Connect URL.                      |
+| platform_back_dns_name      | `"platform-back.{{ app_dns_domain }}"`   | DNS name use for the Platform back URL.                |
+| platform_front_dns_name     | `"platform.{{ app_dns_domain }}"`        | DNS name use for the Platform URL.                     |
+| iam_dns_name                | `"iam.{{ app_dns_domain }}"`             | DNS name use for the Keycloak (identity provider) URL. |
+| documentation_dns_name      |`"doc.{{ app_dns_domain }}"`              | DNS name use for the documentation URL.                |
```

Comment:
We did. I said we should provide versioned documentation, at least for the on-premises, so that customers can have the documentation for the version they have and not the last version. We concluded it was easier to deploy their own documentation, so they always have the corresponding version.